go/cmd/vtgate/vtgate.go

```diff
@@ -55,6 +61,7 @@ func init() {
 	servenv.RegisterGRPCServerFlags()
 	servenv.RegisterGRPCServerAuthFlags()
 	servenv.RegisterServiceMapFlag()
+	servenv.OnParseFor("vtgate", RegisterFlags)
```

Comment:
nit: because this package is unimport-able, you can use `OnParse` instead of `OnParseFor` (since no other binary can even get here, only "vtgate" will ever be the one getting flags parsed)